### DISCUSSION TITLE
Firefox 118.0.1 hangs on "loading resources"

### DISCUSSION THREAD

**Discussion by robot144** [widgetti/solara#311] [Q&A] [answered]
Hi,
After an update of solara (and possibly some other packages), I cannot get solara to work anymore on my computer running Ubuntu 23.04. The server will start without any warnings. Then it will start a page in my browser (firefox 118.0.1), which then hangs showing the text "Loading resources". The test is run with "solara run sol.py" and the example from here https://solara.dev/docs/tutorial/web-app.
I have tried updating anaconda and various versions of solara through pip and conda-forge. All give the same issue. In the console of the browser there is only one message that may point somewhere "Uncaught (in promise) ReferenceError: for_pyodide is not defined solaraInit http://localhost:8765/static/main-vuetify.js:159"
If I use chrome or an old version of firefox, then the issue does not appear. This gives me a workaround, but it would be nice if the default firefox (which is 118.0.1 since Sept29 on my machine) on ubuntu would also work. Unfortunately, I do not have much background in javascript, so it's hard for me to dig beyond this point.
Best regards,  Martin

**Comment by maartenbreddels** [comment, marked as answer]
Hi Martin,

this could be a cache issue giving your old javascript files. Could you 
 * open the dev console
 * go to the network tab
 * check "disable cache"
 * refresh the page

Or some other way to avoid the cache.

I hope this solves your problem!

Regards,

Maarten

**Reply by robot144** [reply]
Hi Maarten,
Thanks! This the trick for me. This is very useful for me, since messing about with other browsers is much more of a hassle.
Best regards,  Martin
